appintro/src/main/res/layout/appintro_intro_layout2.xml

```diff
@@ -31,7 +31,7 @@
             android:id="@+id/bottom_separator"
             android:layout_width="match_parent"
             android:layout_height="1px"
-            android:background="#55000000"
+            android:background="@color/textBg"
```

Comment:
So this `<TextView>` can be removed entirely in this layout file.
Looks like the `bottom_separator` is used only on `layout` and not in `layout2`.
Can I ask you to apply this fix on the other layout file: `appintro/src/main/res/layout/appintro_intro_layout.xml`?

Reply:
What do you mean with applying this fix to the other layout file ?

Should i make non-hardcoded the color there too or delete the TextView?

Reply:
> What do you mean with applying this fix to the other layout file ?

Sorry so, I'd:
* Remove the `<TextView>` from the `_layout2.xml` file
* Apply the fix (don't hardcode the color) from the `_layout.xml`